pkg/search/controller.go

```diff
@@ -264,11 +264,16 @@ func (c *Controller) updateResourceRegistry(oldObj, newObj interface{}) {
 	oldRR := oldObj.(*v1alpha1.ResourceRegistry)
 	newRR := newObj.(*v1alpha1.ResourceRegistry)
 
+	if reflect.DeepEqual(oldRR.Spec, newRR.Spec) {
+		klog.Infof("spec of resourceRegistry[%s] has not changed", newRR.Name)
```

Comment:
```suggestion
		klog.V(4).Infof("Ignore ResourceRegistry(%s) update event as spec not changed", newRR.Name) 
```

This log can be suppressed or ignored I think. I can't see why we need it.

Reply:
@RainbowMango That's right.  the update event message doesn't make sense. 
I hadn't thought of that，it's good.
Done